types/jest-matcher-utils/index.d.ts

```diff
@@ -0,0 +1,45 @@
+// Type definitions for jest-matcher-utils 21.0
+// Project: http://facebook.github.io/jest/
+// Definitions by: Alex Coles <https://github.com/myabc>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+import * as chalk from 'chalk';
+
+export const EXPECTED_COLOR: chalk.ChalkChain;
+export const RECEIVED_COLOR: chalk.ChalkChain;
+// export const EXPECTED_BG: chalk.ChalkChain; // removed in b430e51a
```

Comment:
N.B. @pedrottimark removed these constants in a commit on `master` (yet to be released) https://github.com/facebook/jest/commit/b430e51a34b389c0f43dfdcf05fab5220a7b5677#diff-98e61ded296e35d3a4fcae207101edeb - so I guess they can be considered de=facto "deprecated"

Reply:
If it is not released, it shouldn't be removed now.

Reply:
@plantain-00 ok, thanks.